mmv1/third_party/terraform/utils/provider.go.erb

```diff
@@ -206,6 +206,7 @@ func Provider() *schema.Provider {
 			},
 		},
 
+		// ####### START datasources ###########
```

Comment:
It'd likely be easier for a tool to parse if the start markers were down a line & end markers up a line, such that the only lines between them are the resource lines. It's pretty marginal since they can just skip a line on either side though 🤷 

i.e.

```
// ####### START datasources ###########
"google_active_folder": dataSourceGoogleActiveFolder(),
...
"google_redis_instance": dataSourceGoogleRedisInstance(),
// ####### END datasources ###########
```